app/src/main/java/com/keylesspalace/tusky/entity/Emoji.kt

```diff
@@ -16,10 +16,14 @@
 package com.keylesspalace.tusky.entity
 
 import android.os.Parcelable
+import android.support.annotation.Nullable
+import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
 data class Emoji(
         val shortcode: String,
-        val url: String
+        val url: String,
+        @Nullable
```

Comment:
Not sure if this is needed, since the type is `Boolean?`, but android studio complained about possible NPE at the call site without it, so...